token/program-2022/src/processor.rs

```diff
@@ -1,5 +1,7 @@
 //! Program state processor
 
+use crate::extension::Extension;
```

Comment:
Can you remove this line? Looks like it isn't used according to clippy

Reply:
Okay. Thanks!